go.mod

```diff
@@ -3,92 +3,98 @@ module github.com/syncthing/syncthing
 go 1.21.0
 
 require (
-	github.com/AudriusButkevicius/recli v0.0.7-0.20220911121932-d000ce8fbf0f
```

Comment:
@emlun You might want to remove these downgrades from this PR, yes?

Reply:
Honestly I have no clue how dependency management works in Go, I've just run `go mod tidy` when instructed by the tools. Merging from `main` and re-running `go mod tidy` seems to have no effect. How do I fix this?

Reply:
Ok, this seems to have worked:

1. Revert the files to their state at the `main` branch: `git checkout main -- go.mod go.sum`
2. Re-run `go mod tidy` (as instructed by the tools when I attempted to rebuild, as expected).